src/core/_buttons.css

```diff
@@ -19,6 +19,11 @@ input[type="button"] {
   border-radius: var(--border-radius);
   border: var(--button-border);
   cursor: pointer;
+  -webkit-transition: border 0.2s;
```

Comment:
The build process uses Autoprefixer. Autoprefixer uses data from http://caniuse.com/ to determine which prefixes are needed based on a set of "target" browsers. 

The default set of target browsers is defined by the Browserlist module as `> 1%, last 2 versions, Firefox ESR`, which roughly translates to "any browser with greater that 1% market share, the newest two versions of all browsers, and the latest Firefox ESR version".

See https://github.com/ai/browserslist for a breakdown of what that string means in more detail.

Manually adding prefixes for browsers is therefore (in theory, anyway) unnecessary.

To customize Skeleton's browser support... Well we probably should make a thing that does that. @gld1982ltd, how do you feel about adding something like a config file to this project that your build process module picks up and uses to tell Autoprefixer what to do?